packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -220,6 +225,21 @@ class CreateCommand extends CreateBase {
     } else if (platforms == null || platforms.isEmpty) {
       throwToolExit('Must specify at least one platform using --platforms',
         exitCode: 2);
+    } else if (generateFfiPlugin &&
+        argResults.wasParsed('platforms') &&
+        platforms.contains('web')) {
+      throwToolExit('The web platform is not supported in plugin_ffi template.',
+          exitCode: 2);
+    } else if (generateFfiPlugin && argResults.wasParsed('ios-language')) {
+      throwToolExit(
+          'The option "ios-language" is not supported with the plugin_ffi '
+          'template: the language will always be C or C++.',
+          exitCode: 2);
+    } else if (generateFfiPlugin && argResults.wasParsed('android-language')) {
+      throwToolExit(
+          'The option "android-language" is not supported with the plugin_ffi '
+          'template: the language will always be C or C++.',
+          exitCode: 2);
```

Comment:
This code might be more readable if indented something like this:

```suggestion
    } else if (generateFfiPlugin && argResults.wasParsed('platforms') && platforms.contains('web')) {
      throwToolExit(
        'The web platform is not supported in plugin_ffi template.',
        exitCode: 2,
      );
    } else if (generateFfiPlugin && argResults.wasParsed('ios-language')) {
      throwToolExit(
        'The option "ios-language" is not supported with the plugin_ffi '
        'template: the language will always be C or C++.',
        exitCode: 2,
      );
    } else if (generateFfiPlugin && argResults.wasParsed('android-language')) {
      throwToolExit(
        'The option "android-language" is not supported with the plugin_ffi '
        'template: the language will always be C or C++.',
        exitCode: 2,
      );
```